test/unit/process/test_openacc_process_set.cpp

```diff
@@ -0,0 +1,9 @@
+#include <gtest/gtest.h>
+
+#include <micm/process/process.hpp>
+#include <micm/process/process_set_openacc.hpp>
+
+TEST(ProcessSet, Deriv)
+{
+  micm::openacc::deriv();
+}
```

Comment:
This seems a sample code and I think we need to write the test code for OpenACC in a separate PR. I would delete this code for this PR.

Reply:
Agreed. Again, it was a thing I added to see if the openacc target was working

Reply:
@qinatan Could you please delete this file?

Reply:
done.